provider/pkg/gen/examples/overlays/helmRelease.md

```diff
@@ -1,2 +1,746 @@
+### Specify Helm Chart Values in File and Code
+
+```typescript
+import * as pulumi from "@pulumi/pulumi";
+import * as k8s from "@pulumi/kubernetes";
+import {FileAsset} from "@pulumi/pulumi/asset";
+
+const release = new k8s.helm.v3.Release("redis", {
+    chart: "redis",
+    repositoryOpts: {
+        repo: "https://charts.bitnami.com/bitnami",
+    },
+    valueYamlFiles: [new FileAsset("./metrics.yml")],
+    values: {
+        cluster: {
+            enabled: true,
+        },
+        rbac: {
+            create: true,
+        }
+    },
+});
+
+// -- Contents of metrics.yml --
+// metrics:
+//     enabled: true
+```
+```python
+import pulumi
+from pulumi_kubernetes.helm.v3 import Release, ReleaseArgs, RepositoryOptsArgs
+
+nginx_ingress = Release(
+    "redis",
+    ReleaseArgs(
+        chart="redis",
+        repository_opts=RepositoryOptsArgs(
+            repo="https://charts.bitnami.com/bitnami",
+        ),
+        value_yaml_files=pulumi.FileAsset("./metrics.yml"),
+        values={
+            cluster: {
+                enabled: true,
+            },
+            rbac: {
+                create: true,
+            }
+        },
+    ),
+)
+
+# -- Contents of metrics.yml --
+# metrics:
+#     enabled: true
+```
+```csharp
+using System.Collections.Generic;
+using Pulumi;
+using Pulumi.Kubernetes.Types.Inputs.Helm.V3;
+using Pulumi.Kubernetes.Helm.V3;
+
+class HelmStack : Stack
+{
+    public HelmStack()
+    {
+        var nginx = new Release("redis", new ReleaseArgs
+        {
+            Chart = "redis",
+            RepositoryOpts = new RepositoryOptsArgs
+            {
+                Repo = "https://charts.bitnami.com/bitnami"
+            },
+            ValueYamlFiles = new FileAsset("./metrics.yml");
+            Values = new InputMap<object>
+            {
+                ["cluster"] = new Dictionary<string,object>
+                {
+                    ["enabled"] = true,
+                },
+                ["rbac"] = new Dictionary<string,object>
+                {
+                    ["create"] = true,
+                }
+            },
+        });
+    }
+}
+
+// -- Contents of metrics.yml --
+// metrics:
+//     enabled: true
+```
+```go
+package main
+
+import (
+	"github.com/pulumi/pulumi-kubernetes/sdk/v3/go/kubernetes/helm/v3"
+	"github.com/pulumi/pulumi-kubernetes/sdk/v3/go/kubernetes/yaml"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+)
+
+func main() {
+	pulumi.Run(func(ctx *pulumi.Context) error {
+		_, err := helm.NewRelease(ctx, "redis", helm.ReleaseArgs{
+			Chart:   pulumi.String("redis"),
+			RepositoryOpts: helm.RepositoryOptsArgs{
+				Repo: pulumi.String("https://charts.helm.sh/stable"),
+			},
+			ValueYamlFiles: pulumi.NewFileAsset("./metrics.yml"),
+			Value: pulumi.Map{
+				"cluster": pulumi.Map{
+                    "enabled": pulumi.Bool(true),
+                },
+                "rbac": pulumi.Map{
+                    "create": pulumi.Bool(true),
+                },
+			},
```

Comment:
Looks like the indentation is wrong here

Reply:
Thanks for pointing out! Fixed now.